subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -1788,10 +1770,7 @@ u32_t ticker_update(u8_t instance_index, u8_t user_id, u8_t ticker_id,
 
 	user = &instance->users[user_id];
 
-	last = user->last + 1;
-	if (last >= user->count_user_op) {
-		last = 0U;
-	}
+	last = INC_MOD(user->last + 1, user->count_user_op);
```

Comment:
why additional + 1, the macro already has it. Same for other changes below

Reply:
You are right. Nice catch!